src/sagas/wallet.js

```diff
@@ -387,7 +387,6 @@ export function* loadTokens() {
  * So we fetch the tokens metadata and store on redux
  */
 export function* fetchTokensMetadata(tokens) {
-  // No tokens to load, set metadata as loaded
```

Comment:
Because:

1) It's not setting "metadata as loaded", it's just returning
2) It's pointless, you can already see that there are no tokens to load as the length is 0